### PR TITLE
feat(client): use the progress indicator component when forking a project

### PR DESCRIPTION
**Pull request by andre-code**
### Description
This PR is to use the progress indicator component when forking a project

for projeck progress indicator

/deploy renku-core=develop renku=tests-new-project-form-changes #persist

### CODE REVIEW

File: client/src/project/new/ProjectNew.present.js [hunk 1]
```diff
@@ -156,14 +136,14 @@ function ForkProjectFooter(props) {
...
-          <FontAwesomeIcon icon={faInfoCircle} />{" "}
-          This operation may take a while. You will be redirected automatically or
-          receive a notification at the end.
-        </p>
-      </Fragment>
+      <ProgressIndicator
+        type={ProgressType.Indeterminate}
+        style={ProgressStyle.Light}
+        title="Forking the project..."
+        description="We've received your project information. This may take a while."
```
Comment (ciyer) [inline client/src/project/new/ProjectNew.present.js:143]:
```suggestion
        description="Project is being created."
```

**Review by ciyer — CHANGES_REQUESTED** [review]
Small change to the label.

File: client/src/project/new/ProjectNew.present.js (hunk 1 above)
Comment (andre-code) [reply client/src/project/new/ProjectNew.present.js]: Updated! :)

**Review by ciyer — CHANGES_REQUESTED** [review]
One small request: I know this isn't part of the PR, but since you are making changes to this code, could we switch the position of the Close and Fork buttons so that it is

    Close   Fork

That is, close on the left, fork on the right?

**Comment by andre-code** [comment]
> One small request: I know this isn't part of the PR, but since you are making changes to this code, could we switch the position of the Close and Fork buttons so that it is
> 
> ```
> Close   Fork
> ```
> 
> That is, close on the left, fork on the right?

yes, position changed.

**Review by ciyer — APPROVED** [review]
👍

**Pull request merged**